pages/platform/private-registry/configure-oidc-provider-authentication/guide.en-au.md

```diff
@@ -0,0 +1,288 @@
+---
+title: 'Configure the authentication via an OIDC provider on an OVHcloud Managed Private Registry'
+excerpt: 'Find out how to configure the authentication via an OIDC provider on an OVHcloud Managed Private Registry'
+updated: 2023-08-25
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
+   background-color: #300A24; 
+   color: #ccc;
+   font-family: monospace;
+   padding: 5px;
+   margin-bottom: 5px;
+ }
+ pre.console code {
+   border: solid 0px transparent;
+   color: #ccc;
+   font-family: monospace !important;
+   font-size: 0.75em;
+ }
+ .small {
```

Comment:
Same

Reply:
It will be used when I will add terraform way :)